### PR TITLE
Break after arrow of fun args of infix ops

### PR DESCRIPTION
**Pull request by jberdine**
when "break-infix-before-func = true" and "break-sequences = true"

Related to #993

### CODE REVIEW

File: test/passing/infix_bind.ml [hunk 1]
```diff
@@ -211,3 +211,15 @@ let _ = match () with A -> ( x |> function y -> y (*  *) ) | B -> ()
 let _ = x |> function y -> ( function _ -> y (* A *) ) (* B *)
 
 let _ = () (* This is needed here to avoid the comment above from moving *)
+
+include struct
+  [@@@ocamlformat "break-sequences = true"]
```
Comment (jberdine) [inline test/passing/infix_bind.ml:216]: Is there a better way to add a test with this config?

**Comment by gpetiot** [comment]
I'm not a fan of adding a new behavior based on the combination of 2 options. I didn't see it suggested in #993 is it something that you would like for your codebase at FB ?

Besides I agree that the `break-infix-before-func` option works only on simple code and is broken otherwise, when we have time we can work on a replacement for that.

**Comment by jberdine** [comment]
I don't have strong views on the config interface, but FWIW the reasoning behind the current setup is to view these bind-like infix ops as fancy sequential compositions, and so to use the break-sequences config. That analogy is pretty loose though. Was just trying not to add more options. :-)

There have been a few minor requests for this sort of thing, so we might set it, not sure yet. There seems to be some divergence though, as `break-infix-before-func = false` has seemed mostly not usable. So I don't know.

**Comment by jberdine** [comment]
After further discussion and consideration, it seems better to change code to use monadic let binders. With that context, it does not seem like a good idea to make the implementation of ocamlformat more complex and its behavior less regular and harder to predict.

**Pull request closed without merging**